addons/sale_project/tests/test_sale_project.py

```diff
@@ -318,16 +318,7 @@ def test_links_with_sale_order_line(self):
         sub_B_second = task_B.child_ids.filtered(lambda sub: sub.name == 'Sub B in second project')
         self.assertEqual(sub_B_second.sale_line_id, sale_order_line_B)
 
-        # [CASE 3] Without project --> no sale order line defined
-        task_B.write({
-            'child_ids': [
-                Command.create({'name': 'Sub B without project'}),
-            ]
-        })
-        sub_B_without = task_B.child_ids.filtered(lambda sub: sub.name == 'Sub B without project')
-        self.assertEqual(sub_B_without.sale_line_id, task_B.sale_line_id)
```

Comment:
Same as above, we would still get a result here, it'd be nice to test for it right ?

Reply:
Same answer +-